source/usage-examples/insertOne.txt

```diff
@@ -31,15 +47,16 @@ Expected Result
 ---------------
 
 After you run the full example, you can find the following inserted
-document in the ``haikus`` collection:
+document in the ``restaurants`` collection:
 
 .. code-block:: json
    :copyable: false
 
    {
      "_id": ObjectId("..."),
-     "title": "Record of a Shriveled Datum",
-     "text": "No bytes, no problem. Inserting a document. In MongoDB"
+     "Name": "8282",
+     "Cuisine": "Korean",
+     "Borough": "Manhattan"
```

Comment:
fields are automatically marshalled with lowercase field names

```suggestion
     "name": "8282",
     "cuisine": "Korean",
     "borough": "Manhattan"
```

Reply:
Thanks for catching this — this one went right past me.